tensorly/datasets/imports.py

```diff
@@ -0,0 +1,16 @@
+"""
+Load example datasets.
+"""
+
+import numpy as np
+from sklearn.utils import Bunch
```

Comment:
@borcuttjahns TensorLy does not have sklearn as a dependency. You can instead [copy the definition of a Bunch](https://github.com/scikit-learn/scikit-learn/blob/0d378913b/sklearn/utils/__init__.py#L83). It sounds like it will be used here.